.github/workflows/r-ci.yml

```diff
@@ -70,6 +77,13 @@ jobs:
       #
       # Please see https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases which
       # is crucial for anyone doing releases of TileDB-SOMA.
+      #
+      # Please edit both files in the same way:
+      # * r-ci.yml
+      # * r-python-interop-testing.yml
```

Comment:
I would recommend farming out the logic to a single file that is then included -- as we do here already with with `python-ci-single.yml` which is included by 'minimal' and 'full'.

Reply:
@eddelbuettel thanks for the recommendation.

I will consider that for follow-on work, which is tracked here:
https://github.com/single-cell-data/TileDB-SOMA/issues/2447